admin/app.js

```diff
@@ -303,10 +303,16 @@ app.use(function (err, req, res, next) {
   // catch CSRF errors and redirect to the previous location
   if (err.code === 'EBADCSRFTOKEN') return res.redirect('back')
 
+  const showError = req.app.get('env') === 'development' || req.user.role === 'HELPDESK'
```

Comment:
this shouldnt be a dev only thing - its for production users.
also, the helpdesk in the spec is just an example.  it could be any type of user

Reply:
It's:
```req.app.get('env') === 'development' || req.user.role === 'HELPDESK'```
So it always shows the error (including the stack trace) in dev and *ALSO* shows it for `HELPDESK` users in production